lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -76,6 +76,13 @@ def munge_boolean(value)
     raise ArgumentError.new "invalid value: #{value}"
   end
 
+  def munge_ensure(value)
+    return 'absent' if value.downcase =~ (/^(absent|disable)$/i)
```

Comment:
But then again, I would prefer for a munge_* to go into the type itself anyway. I think there was a reason we shied away from it and put this in `powershell.rb` but I don't remember it off the top of my head.